packages/cli/src/commands/generate/script/templates/script.js.template

```diff
@@ -0,0 +1,5 @@
+// Import files from your api workspace here
+
+export default async ({db, args}) => {
+  // Your script here...
+}
```

Comment:
The formatting is a bit off here, whilst it will run through prettier, I think we should modify it to match the expected output.
```suggestion
export default async ({ db, args }) => {
  // Your script here...
}
```

Reply:
Updated.